universal-application-tool-0.0.1/app/auth/ProfileFactory.java

```diff
@@ -21,13 +21,21 @@ public ProfileFactory(
   }
 
   public UatProfileData createNewApplicant() {
-    UatProfileData p = new UatProfileData(clock);
-    p.init(dbContext);
-    p.addRole(Roles.ROLE_APPLICANT.toString());
-    return p;
+    return create(Roles.ROLE_APPLICANT);
   }
 
   public UatProfile wrapProfileData(UatProfileData p) {
     return new UatProfile(dbContext, httpContext, p);
   }
+
+  public UatProfileData createNewAdmin() {
```

Comment:
nit: group `createNewAdmin` next to `createNewApplicant`?

Reply:
Done!